dbt_subprojects/dex/models/prices/dex_prices_block_solana.sql

```diff
@@ -0,0 +1,167 @@
+{{ config(
+    schema = 'dex'
+    , alias = 'prices_block_solana'
+    , partition_by = ['block_month']
+    , materialized = 'incremental'
+    , file_format = 'delta'
+    , incremental_strategy = 'merge'
+    , unique_key = ['block_month', 'blockchain', 'contract_address', 'symbol', 'decimals', 'block_number', 'block_time']
+    , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+)
+}}
+
+with dex_trades_raw as (
+    select
+        blockchain
+        , block_number
+        , block_time
+        , token_bought_address
```

Comment:
column names differ a bit in `dex_solana.trades`, you will likely hit errors on this select